test/dynamo/test_dynamo.py

```diff
@@ -632,6 +632,31 @@ def test_all_cpu_tensor(self):
     self.assertIn('MarkStep', met.counter_names())
 
 
+class DynamoOperationsTests(test_utils.XlaTestCase):
+
+  def test_new_with_sizes(self):
+
+    # The addition operation is needed here, since the error only occurs when FakeTensorMode
+    # checks the device of the arguments of some operation. If there's no operation using the
+    # result of Tensor.new, this comparison never occurs.
+    def foo(x):
+      return x.new(*x.size()) + x
+
+    optfoo = torch.compile(backend="openxla")(foo)
```

Comment:
lol is it a new way of calling compile? it used to be `torch.compile(foo, backend="openxla")`